lib/flipper/ui/views/feature.erb

```diff
@@ -293,9 +293,12 @@
                   </div>
                 </div>
                   <div class="col-auto ms-auto">
-                    <button type="button" class="btn btn-outline-secondary" id="add-expression-btn">
-                      Add an expression
-                    </button>
+                    <%# Disabled buttons don't respond to hover events: https://getbootstrap.com/docs/4.0/components/tooltips/#disabled-elements %>
+                    <span class="d-inline-block" tabindex="0" data-toggle="tooltip" data-placement="top" title="Add an expression is only available for 'Any' and 'All' expression types">
```

Comment:
<img width="773" alt="image" src="https://github.com/user-attachments/assets/9658deab-b816-4f0e-bda8-eecb80ac1761" />


This only displays as a native tooltip for me, not as a bootstrap tooltip